### PR TITLE
Add .bashrc recommendation

### PR DESCRIPTION
**Pull request by dspinellis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ Notes:
 
 - Use `alias` to create shortcuts for commonly used commands. For example, `alias ll='ls -latr'` creates a new alias `ll`.
 
+- Save aliases and settings your commonly use in your home directory file named `.bashrc`. This will make them available in all your shell sessions. Synchronize this file among various computers with Git.
```
Comment (neil-greenwood) [inline README.md]: s/settings your/settings you/

**Comment by jlevy** [comment]
Thanks. Agree this should be covered. It may take a little more iteration. In particular, there is always confusion around .profile vs .bashrc however and I think that should be addressed too: http://superuser.com/questions/183870/difference-between-bashrc-and-bash-profile

Finally, any suggestions for where is best to read up on this? I don't think man pages or Stack Exchange are quite right. A link here or in the further reading section is in order.

**Comment by dspinellis** [comment]
I tried to avoid raising this complication, but the issues are difficult to escape.  I pushed a change.

**Comment by jlevy** [comment]
:+1: OK. Open for further refinements, but let's go for it. :)

File: README.md [hunk 2]
```diff
@@ -110,6 +110,12 @@ Notes:
 
 - Use `alias` to create shortcuts for commonly used commands. For example, `alias ll='ls -latr'` creates a new alias `ll`.
 
+- Save aliases, shell settings, and functions you commonly use in `~/.bashrc`, and [arrange for login shells to source it](http://superuser.com/a/183980/7106). This will make your setup available in all your shell sessions.
```
Comment (iveksl2) [inline README.md]: :+1:

**Pull request merged**